packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js

```diff
@@ -274,3 +286,55 @@ export const ArnDescription = () => {
         </TextList>
     </TextContent>);
 };
+
+export const IncludeAliasesLabel = () => {
+    const intl = useIntl();
+
+    return (<React.Fragment>
+        { intl.formatMessage({
+            id: 'cost.arn.include_aliases',
+            defaultMessage: 'Include AWS account aliases'
+        }) }
+        <span onClick={e => e.preventDefault()}>
```

Comment:
Why is this handler here?

Reply:
This is needed to avoid checking the checkbox when clicking the popover but otherwise yep it's unnecessary.